From c00067b227200090bae230f349aff00cbbe3a41f Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 4 May 2016 14:25:35 +0800
Subject: [PATCH 1047/5242] MLK-12748-1 ARM: dts: imx7d: correct usdhc1 cd pin
 setting

commit  e9bf592c7a5ecfe18cde287d63de97003ca398ff from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX7D 19x19 LPDDR2 ARM2 board's uSDHC1 CD pin should be
LOW active, correct it.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts b/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts
index 5573afb..9234b16 100644
--- a/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-19x19-lpddr2-arm2.dts
@@ -444,8 +444,8 @@
 &usdhc1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc1>;
-	cd-gpios = <&gpio5 0 0>;
-	wp-gpios = <&gpio5 1 0>;
+	cd-gpios = <&gpio5 0 GPIO_ACTIVE_LOW>;
+	wp-gpios = <&gpio5 1 GPIO_ACTIVE_HIGH>;
 	no-1-8-v;
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
-- 
1.7.9.5

